components/library/msgpack/Makefile

```diff
@@ -10,25 +10,26 @@
 
 #
 # Copyright 2022-2023 Niklas Poslovski
+# Copyright 2025 Klaus Ziegler
 #
 
 BUILD_STYLE=cmake
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=         msgpack
-COMPONENT_VERSION=      6.0.0
-#COMPONENT_REVISION=     1
+COMPONENT_VERSION=      7.0.0
 COMPONENT_PROJECT_URL=  https://msgpack.org
 COMPONENT_SUMMARY=      An efficient object serialization library
 COMPONENT_FMRI=         library/msgpack
-COMPONENT_SRC=          $(COMPONENT_NAME)-c-$(COMPONENT_VERSION)
+COMPONENT_SRC=          $(COMPONENT_NAME)-cxx-$(COMPONENT_VERSION)
```

Comment:
It looks like you are switching from msgpack C to msgpack C++.  I do not know msgpack internals, but it looks like these are two distinct products and should be packaged separately.

The latest version of msgpack C is 6.1.0.